structure_tests/structure_test_v1.go

```diff
@@ -186,7 +186,21 @@ func ProcessCommand(t *testing.T, envVars []EnvVar, fullCommand []string, checkO
 		} else {
 			t.Fatalf("Error running setup/teardown command: %s.", err)
 		}
-		exitCode = err.(*exec.ExitError).Sys().(syscall.WaitStatus).ExitStatus()
+
+		exitErr, ok := err.(*exec.ExitError)
+		if !ok {
+			exitErr, ok := err.(*exec.Error)
+			if ok {
+				t.Logf("Command %s failed to run! Error: %s", exitErr.Name, exitErr.Error())
+				t.Logf("Attempting to run in shell mode.")
```

Comment:
Hmm I'm not a huge fan of trying to run the command twice. Is there a way to tell if we need to run it in shell mode? Or could we have users specify it as a bool?